src/components/Navbar.js

```diff
@@ -1,71 +1,97 @@
+function Navbar({ profLoggedIn = false }) {
+	const [click, setClick] = useState(false);
+	const [button, setButton] = useState(true);
 
-  const handleClick = () => setClick(!click);
-  const closeMobileMenu = () => setClick(false);
-  const showButton = () => {
-    if (window.innerWidth <= 960) setButton(false);
-    else setButton(true);
-  };
-  useEffect(() => {
-    showButton();
-  }, []);
-  window.addEventListener("resize", showButton);
+	const handleClick = () => setClick(!click);
+	const closeMobileMenu = () => setClick(false);
+	const showButton = () => {
+		if (window.innerWidth <= 960) setButton(false);
+		else setButton(true);
+	};
+	useEffect(() => {
+		showButton();
+	}, []);
+	window.addEventListener("resize", showButton);
 
-  return (
-    <nav className="navbar">
-      <div className="navbar-container">
-        <Link to="/" className="navbar-logo" onClick={closeMobileMenu}>
-          Check My Attendance
-          <FontAwesomeIcon className="fa-shekel-sign" icon={faShekelSign} />
-        </Link>
-        <div className="menu-icon" onClick={handleClick}>
-          <FontAwesomeIcon
-            icon={click ? faTimes : faBars}
-            className={click ? "fas fa-times" : "fas fa-bars"}
-          />
-        </div>
-        <ul className={click ? "nav-menu active" : "nav-menu"}>
-          <li className="nav-item">
-            <Link to="/" className="nav-links" onClick={closeMobileMenu}>
-              Home
-            </Link>
-          </li>
-          <li className="nav-item">
-            <Link
-              to="/about-us"
-              className="nav-links"
-              onClick={closeMobileMenu}
-            >
-              About Us
-            </Link>
-          </li>
-          <li className="nav-item button">
-            <Link
-              to="/sign-in"
-              className="nav-links-mobile"
-              onClick={closeMobileMenu}
-            >
-              Sign In
-            </Link>
-          </li>
-        </ul>
-        {button && <Button buttonStyle={"btn--outline"}>SIGN IN</Button>}
-      </div>
-    </nav>
-  );
+	return (
+		<nav className="navbar-nav">
+			<div className="navbar-container">
+				<Link to="/" className="navbar-logo" onClick={closeMobileMenu}>
+					Check My Attendance
+					<FontAwesomeIcon
+						className="fa-shekel-sign"
+						icon={faShekelSign}
+					/>
+				</Link>
+				<div className="menu-icon" onClick={handleClick}>
+					<FontAwesomeIcon
+						icon={click ? faTimes : faBars}
+						className={click ? "fas fa-times" : "fas fa-bars"}
+					/>
+				</div>
+				<ul className={click ? "nav-nav-menu active" : "nav-nav-menu"}>
+					<li className="nav-item">
+						<Link
+							to="/"
+							className="nav-links"
+							onClick={closeMobileMenu}
+						>
+							Home
+						</Link>
+					</li>
+					<li className="nav-item">
+						<Link
+							to="/student"
+							className="nav-links"
+							onClick={closeMobileMenu}
+						>
+							Student Sign In
```

Comment:
I think having one button for singing in is better than two individual ones. Maybe once inside the sign in page, the user can choose whether they want to sing in as a prof or user. I can take a look at this.

Reply:
fixed
![image](https://github.com/diegobajetti/check-my-attendance/assets/49965519/3f488c0d-a5a7-4660-81ee-48cd84c21284)